packages/opal-common/opal_common/fetcher/providers/http_fetch_provider.py

```diff
@@ -73,7 +73,10 @@ async def __aenter__(self):
         if self._event.config.headers is not None:
             headers = self._event.config.headers
         if opal_common_config.HTTP_FETCHER_PROVIDER_CLIENT == "httpx":
-            self._session = httpx.AsyncClient(headers=headers)
+            timeout = opal_common_config.HTTPX_FETCHER_TIMEOUT
+            if timeout is None:
+                timeout = 5
```

Comment:
the default timeout in httpx is already 5 ( [ref](https://github.com/encode/httpx/blob/9e8ab40369bd3ec2cc8bff37ab79bf5769c8b00f/httpx/_config.py#L246) )

Reply:
anyway, can be simplified as timeout = opal_common_config.HTTPX_FETCHER_TIMEOUT or 5.0 but I think a better approach is to make a default to the config instead of that and make it strict instead of optional